CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 6.2.2 (in progress)
 
 * Your contribution here!
+* [#2097](https://github.com/oshi/oshi/pull/2097): Fixed Issue #2097, by updating regEx - [@varun83388](https://github.com/varun83388).
```

Comment:
This should be the number of your PR, 2108.

Reply:
Also I liked the more detailed description message in your earlier commit, that matched the issue title.

Reply:
I've made the changes, are you able to see it?

Reply:
> Also I liked the more detailed description message in your earlier commit, that matched the issue title.

Thank you for being patient, really motivates the first-timers! :)

Reply:
I can see the new number but still the old description message.  Referencing another issue doesn't really tell users what changed.  In your #2107 version you used "Prefer character classes to alternators in regex" which is good.